aws/aws-how-to/eks-gpu-enablement.rst

```diff
@@ -0,0 +1,115 @@
+Enable GPU on Ubuntu EKS
+
+This guide will give step-by-step of how to enable GPU on your EKS worker nodes.
+
+Before following this instruction, you should create cluster and nodegroups with GPU support instance types and have SSH access to nodes.
+
+==============================================
+
+GPU supported instance types
+--------------------------
+You can see detailed instance description `here <https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html>`.
+
+
+Install nvidia ubuntu driver on nodes
+--------------------------
+First, update package index files
+.. code-block:: bash
+
+   $ sudo apt-get update
+
+To enable GPU, you need to install nvidia driver. There are `multiple different ways <https://help.ubuntu.com/community/NvidiaDriversInstallation>` to install driver but you can simply use `ubuntu-drivers`.
```

Comment:
We should suggest the official drivers supported by the target platform (i.e. the right instance type). See [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html).

Reply:
Just expanding on this - we don't need to have a step by step guide on driver installation since its covered by the AWS docs. Lets link to the official documentation and recommend using the Tesla driver for general purpose GPU uses.